cmd/flag/correct_order_flags.go

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright © 2019 Infostellar, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package flag
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/infostellarinc/go-stellarstation/api/v1"
+)
+
+var (
+	// Default correct order.
+	defaultCorrectOrder = false
+
+	defaultBundleCountThreshold     = 20
+	defaultBundleByteThreshold  int = 10e6 // 1M
+	defaultDelayThreshold           = 500  // Millisecond
+)
+
+type CorrectOrderFlags struct {
+	CorrectOrder bool
+
+	BundleCountThreshold int
+	BundleByteThreshold  int
+	DelayThreshold       int
+}
+
+// Add flags to the command.
+func (f *CorrectOrderFlags) AddFlags(cmd *cobra.Command) {
+	cmd.Flags().IntVarP(&f.DelayThreshold, "delay-threshold", "", defaultDelayThreshold,
+		"The maximum amount of time in milliseconds that packets remain in the sorting pool.")
```

Comment:
@vaelen Could I ask you to check grammar and wording of each usage text?

Reply:
Your grammar is correct.